### ISSUE TITLE
Different mAP with same model on TF OD API mrcnn and Matterport mrcnn

### ISSUE THREAD

**Issue by silvernox-dev** [matterport/Mask_RCNN#2426]
The mask mAPs are -
Using Eval functions of Matterport repo - mAP@0.5IOU - 0.15
Using eval.py of TF OD API MRCNN - mAP@0.5IOU - 0.21 (using pascal_voc_instance_segmentation_metrics)

One model is being evaluated using the eval scripts/functions of the above two methods. Was wondering what is different between Keras and Tensorflow implementations?
